carsus/io/output/base.py

```diff
@@ -106,6 +112,98 @@ def __init__(self,
 
         logger.info('Finished.')
 
+    @classmethod
+    def from_config(cls, fname):
+        with open(fname) as stream:
+            loaded_config = yaml.safe_load(stream)
+            loaded_config_keys = loaded_config.keys()
+
+            nist_config = loaded_config["nist"]
+            if nist_config["atomic_weights"] == "default":
+                atomic_weights = NISTWeightsComp()
+            else:
+                atomic_weights = NISTWeightsComp(
+                    nist_config["atomic_weights"]
+                )
+            ionization_energies = NISTIonizationEnergies(
+                nist_config["ionization_energies"]
+            )
+
+            gfall_config = loaded_config["gfall"] 
+            if gfall_config["path"] == "default":
+                # TODO: better alternative?
```

Comment:
This is not necessary. In `carsus/io/kurucz/gfall.py` line 54 you can see `fname=None` and then the reader downloads the file. So default should be `None`.